.github/actions/fastlane-android/action.yaml

```diff
@@ -29,35 +30,38 @@ inputs:
     required: false
     default: ""
   java_distribution:
-    description: Option of action/setup-java@v2
+    description: Option of action/setup-java@v4
     required: false
     default: 'adopt'
   java_version:
-    description: Option of action/setup-java@v2
+    description: Option of action/setup-java@v4
     required: false
     default: '11'
 runs:
   using: "composite"
   steps:
-    # Setup ruby on github hosted machines, step fails on self-hosted macjenkins
+    # Setup step fails on self-hosted macjenkins
     - name: Setup ruby
-      if: ${{ runner.name == 'Hosted Agent' }}
+      if: ${{ runner.platform != 'darwin' }}
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: ${{ inputs.ruby_version }}
         bundler-cache: true
     - name: Setup java
-      uses: actions/setup-java@v2
+      uses: actions/setup-java@v4
       with:
         distribution: ${{ inputs.java_distribution }}
         java-version: ${{ inputs.java_version }}
+    - name: Setup Android SDK
+      uses: android-actions/setup-android@v3
```

Comment:
So, this action install `sdkmanager`, will Fastlane install necessary platform-tools and etc by itself?

Reply:
Yes
```shell
[09:13:22]: --- Step: assembledevRelease ---
[09:13:22]: --------------------------------
[09:13:22]: $ /home/runner/_work/focus-medical-apapp-frontend/focus-medical-apapp-frontend/android/gradlew assembledevRelease -p android -PapplicationId=com.focusmedical.apapp.app.develop -Pandroid.injected.signing.store.file=/home/runner/_work/focus-medical-apapp-frontend/focus-medical-apapp-frontend/fastlane/release.keystore -Pandroid.injected.signing.store.*** -Pandroid.injected.signing.key.alias=upload -Pandroid.injected.signing.key.*** -PversionCode=1 -PversionName=''
[09:13:22]: ▸ Downloading https://services.gradle.org/distributions/gradle-8.0.2-all.zip
[09:13:25]: ▸ ...............10%................20%................30%................40%................50%................60%................70%................80%................90%................100%
[09:13:29]: ▸ Welcome to Gradle 8.0.2!
[09:13:29]: ▸ Here are the highlights of this release:
[09:13:29]: ▸ - Improvements to the Kotlin DSL
[09:13:29]: ▸ - Fine-grained parallelism from the first build with configuration cache
[09:13:29]: ▸ - Configurable Gradle user home cache cleanup
[09:13:29]: ▸ For more details see https://docs.gradle.org/8.0.2/release-notes.html
[09:13:30]: ▸ Starting a Gradle Daemon (subsequent builds will be faster)
[09:14:07]: ▸ > Configure project :app
[09:14:07]: ▸ WARNING: Using flatDir should be avoided because it doesn't support any meta-data formats.
[09:14:10]: ▸ > Configure project :capacitor-cordova-android-plugins
[09:14:10]: ▸ WARNING: Using flatDir should be avoided because it doesn't support any meta-data formats.
[09:14:12]: ▸ Errors during XML parse:
[09:14:12]: ▸ Additionally, the fallback loader failed to parse the XML.
[09:14:12]: ▸ Checking the license for package Android SDK Build-Tools 30.0.3 in /home/runner/.android/sdk/licenses
[09:14:12]: ▸ License for package Android SDK Build-Tools 30.0.3 accepted.
[09:14:12]: ▸ Preparing "Install Android SDK Build-Tools 30.0.3 (revision: 30.0.3)".
[09:14:15]: ▸ "Install Android SDK Build-Tools 30.0.3 (revision: 30.0.3)" ready.
[09:14:15]: ▸ Installing Android SDK Build-Tools 30.0.3 in /home/runner/.android/sdk/build-tools/30.0.3
[09:14:15]: ▸ "Install Android SDK Build-Tools 30.0.3 (revision: 30.0.3)" complete.
[09:14:15]: ▸ "Install Android SDK Build-Tools 30.0.3 (revision: 30.0.3)" finished.
[09:14:15]: ▸ Checking the license for package Android SDK Platform 33 in /home/runner/.android/sdk/licenses
[09:14:15]: ▸ License for package Android SDK Platform 33 accepted.
[09:14:15]: ▸ Preparing "Install Android SDK Platform 33 (revision: 3)".
[09:14:19]: ▸ "Install Android SDK Platform 33 (revision: 3)" ready.
[09:14:19]: ▸ Installing Android SDK Platform 33 in /home/runner/.android/sdk/platforms/android-33
[09:14:19]: ▸ "Install Android SDK Platform 33 (revision: 3)" complete.
[09:14:19]: ▸ "Install Android SDK Platform 33 (revision: 3)" finished.
```